src/main/scala/org/hammerlab/guacamole/loci/map/LociMap.scala

```diff
@@ -99,6 +99,19 @@ object LociMap {
   }
 
   /**
+   * Build a map from some LociSets, mapping each set's loci to that sets index in `lociSets`.
+   */
+  def apply(lociSets: Iterable[LociSet]): LociMap[PartitionIndex] = {
```

Comment:
Function looks fine, but might be a bit strange to have LociMap use stuff from `loci.partitioning.LociPartitioner`. Maybe just have it return a `LociMap[Int]`? Also this `apply` seems kind of niche to loci partitioning and not something that it's obvious from the signature what it does, so could be worth giving it a descriptive name